envs/mamba-env.yaml

```diff
@@ -11,20 +11,20 @@ packages:
     git_tag : 8eb554c81192737f0bdf4369649fd3a36a64d69b	
   - feedstock : https://github.com/conda-forge/nlohmann_json-feedstock.git
     git_tag : 2890d321829957c5064dc34fd26ab552e52ed84f
-  - feedstock : https://github.com/conda-forge/reproc-feedstock.git
-    git_tag : 84330a3a928f0be8bfe360e28529111ddba89aa1
-  - feedstock : https://github.com/conda-forge/libiconv-feedstock.git
-    git_tag : 0b738dfd8cfa3b5072b00a62a7cb41b21e018be5
-    patches :
-          - feedstock-patches/libiconv/libiconv.patch
-  - feedstock : https://github.com/conda-forge/libsolv-feedstock.git
-    git_tag : a4377e7bb252838fe3b2595a976a4f1b9a60341b
-  - feedstock : https://github.com/conda-forge/spdlog-feedstock.git
-    git_tag: 6f6a8b720f06142b20e03fe7236a79ea5b594728
-    patches:
-          - feedstock-patches/spdlog/0001-disable-systemd-tests-to-fix-baremetal-build.patch
-  - feedstock : https://github.com/conda-forge/termcolor-cpp-feedstock.git
-    git_tag : f35835967df63a49285fb70ea08edd704ab0aa79	
+#  - feedstock : https://github.com/conda-forge/reproc-feedstock.git
+#    git_tag : 84330a3a928f0be8bfe360e28529111ddba89aa1
```

Comment:
Can we delete these lines instead of commenting? Also delete the patches that were being referred here.